api/turing/cluster/controller_test.go

```diff
@@ -99,6 +100,7 @@ func TestDeployKnativeService(t *testing.T) {
 	// Define tests
 	cs := knservingclientset.NewSimpleClientset()
 	tests := map[string][]reactor{
+		// TODO: problematic for new_service
```

Comment:
What does it mean?

Reply:
Oh this was supposed to be removed, I made a note that this test was failing at first but I fixed it.